vaadin-master-detail-layout-flow-parent/vaadin-master-detail-layout-flow/src/main/java/com/vaadin/flow/component/masterdetaillayout/MasterDetailLayout.java

```diff
@@ -173,7 +188,8 @@ public String getMasterSize() {
      * Sets the size of the master area in CSS length units. When specified, it
      * prevents the master area from growing or shrinking. If there is not
      * enough space to show master and detail areas next to each other, the
-     * layout switches to the overlay mode.
+     * layout switches to the overlay, using the mode defined by
+     * {@link #setOverlayMode(OverlayMode)}.
```

Comment:
```suggestion
     * enough space to show master and detail areas next to each other, the
     * details are shown in an overlay, using the mode defined by
     * {@link #setOverlayMode(OverlayMode)}.
```
We could make it more explicit what is shown in the overlay.

Reply:
Thanks, done.